requirements-test.txt

```diff
@@ -1,3 +1,5 @@
+-r requirements-base.txt
```

Comment:
I think the general idea here is:
- when you need to simply install a package, you install everything from requirements.txt
- when you want to install test-related packages, you install it **additionally**, with separate `pip install -r` cmd

So for me it's a matter of taste and I don't see any strong reason to change it.
Do you agree or I'm missing anything else here?